src/main/java/org/broadinstitute/dsm/db/Participant.java

```diff
@@ -158,9 +186,10 @@ public static Participant getParticipant(@NonNull Map<String, Assignee> assignee
                 assigneeTissue = assignees.get(assigneeIdTissue).getName();
             }
         }
-        Participant participant = new Participant(rs.getString(DBConstants.PARTICIPANT_ID),
+        Participant participant = new Participant(rs.getLong(DBConstants.PARTICIPANT_ID),
                 rs.getString(DBConstants.DDP_PARTICIPANT_ID),
-                assigneeMR, assigneeTissue, realm,
+                rs.getLong(DBConstants.ASSIGNEE_ID_MR),
+                rs.getLong(DBConstants.ASSIGNEE_ID_TISSUE), realm,
```

Comment:
the assigneeMR and assigneeTissue were names before and are now ids, can we change it back to names?

Reply:
@gmakharat @Anzori-Ghurtchumelia